Dockerfile

```diff
@@ -4,13 +4,22 @@
 # that running the image will execute benchmarks with a "warm cache". For
 # example:
 #
-#   docker build --build-arg ETH_RPC_URL=... -t sim .
-#   docker run sim benchmark-hardhat
+#   docker build --build-arg -t sim .
```

Comment:
Was this intended to be changed? Running `docker build --build-arg -t sim .` gave the below error, but the original command still worked

```
"docker build" requires exactly 1 argument.
See 'docker build --help'.

Usage:  docker build [OPTIONS] PATH | URL | -

Build an image from a Dockerfile
```

Also, where does the RPC URL come from if we no longer pass it in a build arg?

Reply:
Ugh. I'm sorry I haven't done a better job of verifying these example commands.  I've now added a simple test script for this Dockerfile, in https://github.com/mds1/convex-shutdown-simulation/pull/8/commits/ef583aedd72191bdd70021e38e04b477e4f3226d . It would be nice if a CI system could run it one day :smile: 

Yes, I intended to remove that `--build-arg` argument, and I did it in https://github.com/mds1/convex-shutdown-simulation/pull/8/commits/471235fe91784349a0e1fb009d54f9841ace133d .  `--build-arg` should be followed by a `KEY=VALUE` pair, and without `ETH_RPC_URL=...` it's nonsensical.

The original command still worked because you're always free to supply extra build arguments, whether the Dockerfile actually refers to them or not.

RPC URL comes from your `.env` file, just like for the non-Docker use cases. Now that we're building the image from the local git Working Tree (as of https://github.com/mds1/convex-shutdown-simulation/pull/8/commits/89046279ee0f30be507351ad9ae57ac77b5f3a2c) the local `.env` file will get copied into the image when it's built, and it will be found at run time just like when running outside of a container.